src/diffpy/utils/tools.py

```diff
@@ -71,16 +71,16 @@ def get_user_info(owner_name=None, owner_email=None, owner_orcid=None):
 
     Parameters
     ----------
-    owner_name: string, optional, default is the value stored in the global or local config file.
+    owner_name : str, optional, default is the value stored in the global or local config file.
         The name of the user who will show as owner in the metadata that is stored with the data
-    owner_email: string, optional, default is the value stored in the global or local config file.
+    owner_email : str, optional, default is the value stored in the global or local config file.
         The email of the user/owner
-    owner_orcid:  string, optional, default is the value stored in the global or local config file.
+    owner_orcid : str, optional, default is the value stored in the global or local config file.
         The ORCID id of the user/owner
 
     Returns
     -------
-    dict:
+    user_info : dict
```

Comment:
seems like numpy uses both (1) what's returned and (2) its type

<img width="644" alt="Screenshot 2024-12-30 at 10 11 02 PM" src="https://github.com/user-attachments/assets/9dd39ef7-f398-480d-ba2e-a0b2756f2d4d" />

Reply:
https://numpydoc.readthedocs.io/en/latest/format.html#returns